docs/reference/frames.md

```diff
@@ -63,3 +63,54 @@ Works like the basic frame, but the content is loaded from a remote `src` first.
   </form>
 </turbo-frame>
 ```
+
+# Attributes and properties
+
+The `<turbo-frame>` element is a [custom element][] with its own set of HTML
+attributes and JavaScript properties.
+
+[custom element]: https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements
+
+## HTML Attributes
+
+* `src` accepts a URL or path value that controls navigation
+  of the element
+
+* `loading` has two valid [enumerated][] values: "eager" and "lazy". When
+  `loading="eager"`, changes to the `src` attribute will immediately navigate
+  the element. When `loading="lazy"`, changes to the `src` attribute will defer
+  navigation until the element is visible in the viewport.
+
+* `busy` is a [boolean attribute][] toggled to be present when a
+  `<turbo-frame>`-initiated request starts, and toggled false when the request
+  ends
+
+* `disabled` is a [boolean attribute][] that prevents any navigation when
+  present
+
+* `target` refers to another `<turbo-frame>` element by ID to be navigated when
+  a descendant `<a>` is clicked. When `target="_top"`, navigate the window.
+
+* `recurse`
```

Comment:
Thanks for merging this @dhh. I didn't know enough about the specifics of `recurse`, `autoscroll` or `data-autoscroll-block` to elaborate further. Maybe @sstephenson would be able to augment these changes.

Reply:
I've fixed some broken link syntax and done my best to describe `autoscroll` and `data-autoscroll-block` in https://github.com/hotwired/turbo-site/pull/36.